admin/system/insert-produk.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+    include("connection.php");
+    session_start();
+
+    //get data
+    $nama_produk = $_POST["nama_produk"];
+    $jenis_produk = $_POST["jenis_produk"];
+    $id = uniqid();
+    mkdir("../upload/".$id);
+
+    if(isset($_POST["kategori"])){
+        $kategori = $_POST["kategori"];
+    }
+    else{
+        $kategori = null;
+    }
+
+    if(isset($_FILES["gambar"]) || isset($_POST["jumlah_gambar"])){
+        if(isset($_FILES["gambar"])){
+            mkdir("../upload/".$id."/gambar");
+            $target = "1.".strtolower(pathinfo(basename($_FILES["gambar"]["name"]),PATHINFO_EXTENSION));
+            $target_dir = "../upload/".$id."/gambar/";
+            $target_file = $target_dir . $target;
+            $imageFileType = strtolower(pathinfo($target_file,PATHINFO_EXTENSION));
+            
+            if (move_uploaded_file($_FILES["gambar"]["tmp_name"], $target_file)) {
```

Comment:
Codacy found an issue: [Direct use of $_FILES Superglobal detected.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)